### PR TITLE
F/clean up

### PR DESCRIPTION
**Pull request by Kayra**

### CODE REVIEW

**Comment by RobertDickens** [comment]
@Kayra rewrote some descriptions:
Regularised regression:"Regularisation techniques to simplify models to avoid overfitting and improve interpretability."
Linear Regression: "Deriving the gradient descent algorithm to fit a linear regression model and implementing it in python."
Logistic Regression: ""Implementing a linear model for classification tasks in Python"
